.travis.yml

```diff
@@ -84,42 +84,42 @@ matrix:
     - jdk: oraclejdk8
       env:
         - DESC="tests de"
-        - CMD="mvn -X clean integration-test failsafe:verify -DargLine='-Duser.language=de -Duser.country=DE'"
+        - CMD="mvn -X clean integration-test failsafe:verify -DargLine='-Duser.language=de -Duser.country=DE -Xms1024m -Xmx2048m'"
     # unit tests in Spanish locale (oraclejdk8)
     - jdk: oraclejdk8
       env:
         - DESC="tests es"
-        - CMD="mvn -X clean integration-test failsafe:verify -DargLine='-Duser.language=es -Duser.country=ES'"
+        - CMD="mvn -X clean integration-test failsafe:verify -DargLine='-Duser.language=es -Duser.country=ES -Xms1024m -Xmx2048m'"
     # unit tests in Finnish locale (oraclejdk8)
     - jdk: oraclejdk8
       env:
         - DESC="tests fi"
-        - CMD="mvn -X clean integration-test failsafe:verify -DargLine='-Duser.language=fi -Duser.country=FI'"
+        - CMD="mvn -X clean integration-test failsafe:verify -DargLine='-Duser.language=fi -Duser.country=FI -Xms1024m -Xmx2048m'"
     # unit tests in French locale (oraclejdk8)
     - jdk: oraclejdk8
       env:
         - DESC="tests fr"
-        - CMD="mvn -X clean integration-test failsafe:verify -DargLine='-Duser.language=fr -Duser.country=FR'"
+        - CMD="mvn -X clean integration-test failsafe:verify -DargLine='-Duser.language=fr -Duser.country=FR -Xms1024m -Xmx2048m'"
     # unit tests in Chinese locale (oraclejdk8)
     - jdk: oraclejdk8
       env:
         - DESC="tests zh"
-        - CMD="mvn -X clean integration-test failsafe:verify -DargLine='-Duser.language=zh -Duser.country=CN'"
+        - CMD="mvn -X clean integration-test failsafe:verify -DargLine='-Duser.language=zh -Duser.country=CN -Xms1024m -Xmx2048m'"
     # unit tests in Japanese locale (oraclejdk8)
     - jdk: oraclejdk8
       env:
         - DESC="tests ja"
-        - CMD="mvn -X clean integration-test failsafe:verify -DargLine='-Duser.language=ja -Duser.country=JP'"
+        - CMD="mvn -X clean integration-test failsafe:verify -DargLine='-Duser.language=ja -Duser.country=JP -Xms1024m -Xmx2048m'"
     # unit tests in Portuguese locale (oraclejdk8)
     - jdk: oraclejdk8
       env:
         - DESC="tests pt"
-        - CMD="mvn -X clean integration-test failsafe:verify -DargLine='-Duser.language=pt -Duser.country=PT'"
+        - CMD="mvn -X clean integration-test failsafe:verify -DargLine='-Duser.language=pt -Duser.country=PT -Xms1024m -Xmx2048m'"
     # unit tests in Turkish locale (oraclejdk8)
     - jdk: oraclejdk8
       env:
         - DESC="tests tr"
-        - CMD="mvn -X clean integration-test failsafe:verify -DargLine='-Duser.language=tr -Duser.country=TR'"
+        - CMD="mvn -X clean integration-test failsafe:verify -DargLine='-Duser.language=tr -Duser.country=TR -Xms1024m -Xmx2048m'"
```

Comment:
Instead of recreating the whole argLine, can we have a new property like `argLineExtra` and in the POM, append it to `argLine`?